### PR TITLE
[Woo POS payments onboarding] Show payments IPP onboarding

### LINKED ISSUE #12882: [Woo POS - payments onboarding] Show payments onboarding from existing design

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #12882

### Description

The PR:
* Makes small UI adjustments to make the IPP onboarding error screen look ok
* Handles back icon click there
* Handles the "skip" clicks on the onboarding screens

### Steps to reproduce

Try different cases available to you of the IPP onboarding not finished
* More -> POS
* Try to collect a payment or connect a reader
* Notice that onboarding screens are shown

### The tests that have been performed

I tested:
* WooPAy in test mode
* Cash on delivery disabled
* WooPay disabled (with code modification as with current implementation we won't show POS at all - this is currently in the discussions)

### Images/gif

https://github.com/user-attachments/assets/e265de7c-2445-49f1-9c8a-1a2c3b6faa17

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:

- [ ] The PR is small and has a clear, single focus, or a valid explanation is provided in the description. If needed, please request to split it into smaller PRs.
- [ ] Ensure Adequate Unit Test Coverage: The changes are reasonably covered by unit tests or an explanation is provided in the PR description.
- [ ] Manual Testing: The author listed all the tests they ran, including smoke tests when needed (e.g., for refactorings). The reviewer confirmed that the PR works as expected on big (tablet) and small (phone) in case of UI changes, and no regressions are added.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WooCommerce/src/main/res/values/themes.xml [hunk 1]
```diff
@@ -158,9 +158,8 @@
     <style name="Theme.Woo.Transparent" parent="Theme.Woo">
         <item name="android:windowIsTranslucent">true</item>
         <item name="android:windowBackground">@android:color/transparent</item>
-        <item name="android:windowContentOverlay">@null</item>
         <item name="android:windowNoTitle">true</item>
-        <item name="android:windowIsFloating">true</item>
-        <item name="android:backgroundDimEnabled">false</item>
+        <item name="android:windowIsFloating">false</item>
```
Comment (kidinov) [inline WooCommerce/src/main/res/values/themes.xml:162]: We were not diming the background on all the IPP card reader operations due to simplified implementation for the DM. Now when I fixed the bottom margin for the case when gesture navigation used and changed appearance animation we can change that

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/woocommerce/woocommerce-android/milestone/244">21.2</a>. This milestone is due in less than 2 days.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by samiuelson — APPROVED** [review]
LGTM :shipit:

**Pull request merged**